src/lib/openapi/spec/patch-feature-operation-schema.ts

```diff
@@ -0,0 +1,178 @@
+import { createSchemaObject, CreateSchemaType } from '../types';
+import { featureSchema } from './feature-schema';
+
+//Add operation
```

Comment:
Could probably remove these short comments, just to match the rest of the code base.